include/ideep/lru_cache.hpp

```diff
@@ -212,6 +212,20 @@ class computation_cache {
     return t_store_;
   }
 };
+
+struct dummy_lru {
+  void clear() {};
+};
+
+// Dummy cache to be used where computation cache
+// isn't required but expected. Missing methods should
+// be added as required
+class dummy_computation_cache {
+ public:
+  static inline dummy_lru t_store() {
+    return dummy_lru();
```

Comment:
Returning by value? Can we verify that no code is binding by ref or expects similar semantic to real cache.
```suggestion
    static dummy_lru store;
    return store;
```

Reply:
I've changed it to return by reference to stay consistent with the interface